app/src/main/java/de/westnordost/streetcomplete/quests/bike_parking_capacity/AddBikeParkingCapacityForm.kt

```diff
@@ -17,6 +19,7 @@ class AddBikeParkingCapacityForm : AbstractQuestFormAnswerFragment<Int>() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        binding.clarificationText.isGone = !showClarificationText
```

Comment:
I fear this won't work like that :-/

Think about when the fragment is destroyed and recreated. The Android system can destroy the view and later recreate it when it is not in foreground.

I know, the proper way to pass parameters to a fragment is horribly verbose but I guess there is no other proper way to do this.

Reply:
Hmm, bummer. But makes sense.

Fixed in [`04b3dac`](https://github.com/streetcomplete/StreetComplete/pull/4008/commits/04b3dac7e104cb786fccbdfb4dd6394855eb6501).